datafusion-common/src/dfschema.rs

```diff
@@ -402,8 +402,8 @@ impl Display for DFSchema {
     }
 }
 
-/// Provides schema information needed by [Expr] methods such as
-/// [Expr::nullable] and [Expr::data_type].
+/// Provides schema information needed by certain methods of `Expr`
```

Comment:
now that `Expr` is in a crate that is not used by `datafusion-common` I don't see a good way to offer a docklink (until we have published the version to crates.io / docs.rs)

Reply:
Me neither. I spent much time trying to find a correct link of `Expr`